chapters/buffer_device_address.adoc

```diff
@@ -0,0 +1,181 @@
+// Copyright 2024 The Khronos Group, Inc.
+// SPDX-License-Identifier: CC-BY-4.0
+
+// Required for both single-page and combined guide xrefs to work
+ifndef::chapters[:chapters:]
+ifndef::images[:images: images/]
+
+[[buffer-device-address]]
+= Buffer Device Address
+
+Buffer Device Address allows you to have a pointer to the `VkBuffer` in your shaders.
```

Comment:
It's also used in the API for several other things, including some ray tracing commands.